c_tests/tests/noopts.c

```diff
@@ -18,22 +40,24 @@
 #include <string.h>
 #include <yk_testing.h>
 
-int add_them(int x, int y) { return x + y; }
+int add_them(int x, int y) {
+  // At the call-site x and y are constant, so an optimising compiler would try
```

Comment:
Putting the comment here feels weird given that this isn't where the noopt effect happens.

Reply:
But it is where the optimisation is blocked. Removal of the plus operation is blocked.

Reply:
Sorry I wasn't very clear. I don't think this function needs to be broken out. I think it's equivalent to something like:

```
x = NOOPT(...);
y = NOOPT(...);
z = x + y;
```

i.e. breaking the function out isn't necessary (and is confusing).